erpnext/erpnext_integrations/doctype/payment_plan/payment_plan.json

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
Payment plan and Subscription Plans are same. Please add Payment Plan Id to subscription Id. Also add a description for field payment_plan_id.